back-end/src/sciler/handler/handler_test.go

```diff
@@ -67,7 +67,7 @@ func getTestHandler() *Handler {
 	}
 	communicator := communication.NewCommunicator(workingConfig.General.Host,
 		workingConfig.General.Port, []string{"back-end", "test"})
-	return &Handler{workingConfig, communicator}
+	return &Handler{workingConfig, "fake file name", communicator}
```

Comment:
Since we have a "better" implementation of onInstructionMsg(), maybe the TODO of line 505 is part of this merge?

Reply:
made some tests for this